Makefile

```diff
@@ -1,3 +1,10 @@
+# To be bumped by every release
+VERSION_MAJOR ?= 0
+VERSION_MINOR ?= 1
+VERSION_BUILD ?= 7
```

Comment:
We're going to forget to do this.  I'd like a future version of this change to pull the version from a file which effectively is written by the build system.